tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Models/OrderModelTests.cs

```diff
@@ -0,0 +1,366 @@
+﻿using System;
+using System.Linq;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Api.UnitTests.AutoFixture;
+using NHSD.BuyingCatalogue.Ordering.Common.UnitTests.AutoFixture;
```

Comment:
Do we still need `System.Linq` and `NHSD.BuyingCatalogue.Ordering.Common.UnitTests.AutoFixture`?

Reply:
removed